### DISCUSSION TITLE
Conda Version got stuck at 0.30.3

### DISCUSSION THREAD

**Discussion by lordsoffallen** [huggingface/diffusers#9916] [Q&A]
### Describe the bug

Latest package can not be installed via conda. Here is the stuck PR: https://github.com/conda-forge/diffusers-feedstock/pull/71

### Reproduction

`conda install diffusers` installs 0.30.3

### System Info

Ubuntu 22.04, python 3.10

**Comment by sayakpaul** [comment]
This doesn't seem like a bug in the library. So, not sure what we can do here.

**Comment by lordsoffallen** [comment]
I'm not sure what's the cause, if it's purely CI/CD, we can probably move this discussion to conda PR. I put it here for visibility :man_shrugging:

**Comment by sayakpaul** [comment]
Thanks!

**Comment by lordsoffallen** [comment]
@anton-l Could you maybe help out here?
